doc/sphinx/source/themes/neo4j/static/css/neo4j.css

```diff
@@ -190,3 +194,40 @@ div.code-block-caption:hover > a.headerlink {
     display: block;
     padding: 0.25rem 0;
 }
+
+/* lists */
+
+body.sphinx .doc ul li{
+    margin-bottom: 0.5rem;
+}
+
+/* functions */
+
+body.sphinx .doc dl.py.function {
+    padding-top: 2rem;
+    padding: 1rem;
+    /* border-top: 2px dotted #eee; */
+}
+
+/*  deprecated */
+
+body.sphinx .deprecated::after,
+body.sphinx .deprecated > *::after {
+    content: unset;
+}
+
+body.sphinx .doc dl.deprecated,
+body.sphinx .doc div.deprecated {
+    border-left: 4px solid #d53f8c;
+    color: #702459;
+    background-color: #fed7e2;
+}
+
+body.sphinx .doc dl.deprecated {
+    margin-bottom: 0;
+    padding-bottom: 0;
+}
+
+body.sphinx .doc div.deprecated {
+    padding: 1rem;
+}
```

Comment:
It's very in-your-face. Maybe this follows another color shade from docs tooling? If not, I would suggest maybe a little softer color.
![Screenshot 2023-06-16 at 13 28 07](https://github.com/neo4j/graph-data-science-client/assets/2419675/d8ff4c51-2543-495c-96ab-2d952c0dabaa)

I'm thinking about Neo4j Browser's highlighting, which uses a yellow squiggly line:

![Screenshot 2023-06-16 at 13 37 13](https://github.com/neo4j/graph-data-science-client/assets/2419675/7a253992-c440-4295-8558-2e5f7e918796)
![Screenshot 2023-06-16 at 13 37 22](https://github.com/neo4j/graph-data-science-client/assets/2419675/80c4f5da-41e3-4595-b302-2a2088a54e11)

It seems to me desirable that we have a specific color to indicate the concept of "deprecated". I did a search in various docs and found [this](https://neo4j.com/docs/apoc/current/overview/apoc.warmup/apoc.warmup.run/):

![Screenshot 2023-06-16 at 13 40 08](https://github.com/neo4j/graph-data-science-client/assets/2419675/d8ac2c3e-c4dd-4967-8421-c40d6d4650cf)

My experience also tells me a yellowish color seems intuitive. WDYT?

Reply:
How does this look?

<img width="852" alt="Screenshot 2023-06-20 at 09 29 04" src="https://github.com/neo4j/graph-data-science-client/assets/16576682/8246dea7-5283-4bfd-8b72-e2788051b714">

We could as a future update hide the deprecated div content and display it on hover (so the behavior would be like the glossary entries).